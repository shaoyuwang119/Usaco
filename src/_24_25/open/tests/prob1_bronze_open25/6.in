97 94
D
LD
DDD
WLLD
DWWLD
WWLDWD
DDLWLDD
LLDWDDDD
WLDLWDDDD
LDDDDLDDDD
DLLDDDLWLDD
WLLLLLDWWWWD
WWLDLDLDWDWLD
DWLWDDDWDDWDWD
WLLDDDWLLLWDLDD
WDDDLLDWWDWDDWLD
LDLWLWDDWWDDWDDLD
DDLWLWLLLLLLWLLWDD
LDLDWDWDLWWDWLLLLWD
DLDDLWDLWDLLWDDWDDDD
LLDWLWWDWWLWWLLWDDLWD
WDWWWDLLWLDLDDLLDLDDWD
DWDDWLWLLWLDDWDWLLDDWLD
LDDWLDLLLDLDLLDDDLDDWDDD
DWWWLDLWWWDLDDLLLLLDLWLLD
LDLLDDDWDLDDWWWWWDLWDLLDLD
LLLDDLWLWDLLWLWWDDWWWLDLWWD
DDWLLDDDWDLLWWWDLWWLWLWDWDWD
LWLWDWLLDDDDDDWWDWWWLDLDWWLWD
DLLLWDLDLWLWLLWWLLWLWLDWLLDLWD
DLDDLWDLDDDWWDDWWLDDLWLDWLDLDDD
LWLDLDWDLDLWWWWWWWLLDLDLDDLWDLDD
LDDDLLDLLDDDDWLLWLLLLWWLDWLDDLWWD
DWDWLLDDLWDWWDWLDDDWLWWWDDLLLDLDWD
LWDDDWWDWLDDWLLLLWWWWLWDDDDLWLWDDWD
WDLDLWWDDDLWLWDDWLWLDLLDDLLDLWDDLDLD
DLWWDDDDDDLLLLWLDLWLLLWDWWLWDWLLDDDWD
WLWDWWDLLDLDLWWLWLDLDDDWDLWDLDLLWWWLDD
WDDDLWLLDLLWWLDWLWWLDWDDDDLDWDLDLDLLLWD
LWDLLLLLDWDLLLWLLLLWLLWDWWLDWDDDWLDWDLWD
LLLDDWLDDWLLDWLWLLDDLDDLWWDWLDLDDLWLLDWWD
WWDWLDWWWLWLDDDWLLDWLLDDWLLDWDWLLDWWLDLWDD
WLLLLDDWWDWDWWDDLDDLWDWDDDDWDDLWDLDWDDLWLDD
LDLWWWDLDLDLLWLLWLWLDDLLDLDDLLWWLDDDDWLWDDWD
LDWDLDLLDWWDLLDWLLDWDWDWDWLDDDDDDWLWLLDWDDLLD
WDWWDDLWLDDWLWLLDDDDWWWWDWWDDDDWWWLDLDLLLDWLLD
WLWLWLDLWLLLLWLLLWWDDDDWDDWWLDDWWWDLWWLLDDWLDWD
DDLWDWWWLDLDLDWWWWWLLDLDDLWDLWLLLDDDLLLLWLLWWDWD
LWDDDDDWDWDLDDLDDLDWWDLWWWLWDLWWWWWDDWDDWWLWWWDDD
WWDWWWWWLDDDDLDWDLLDWWDWDDDLWWWLWDDDLWDDLWWWDWLWWD
WDWLLDDLWDDWDLWDDLDWWWDWLDLDLLDWDLLDWLLLLDDDWWDDLWD
WWWDLDLLWDWWLDDLLDWDWDDDLWWLWDWLDDWDDWLDWLWDWLWWWDDD
DLWDWLLDWWLLWLLLWWLWDDDLWDLLDLDDWDDWLLLDDWDDWDDWDLDWD
DWDWWWDDWWDWDWLWLWWWLDWLLWWLDWLDLDDLDLLLWWLWWWDLLWLDDD
DWWWLWDDLWDWLWLDDWLDLDWDDWLWWDDLWWDWDDDDDDDWDWWDWWWWDWD
WLDDWLDWLWWDDWDWDWLDDLWLDDDDLDDWWDWLDLLLLDLWWLLWLWWDWWDD
WWLWWDDDDLWDWLDLLDLWLLDWLLWDDWLDDLDWLLDWLWWLDLLWWLDWWLWLD
WWLLWWDLLWWWDWDDLDLLDWLDDWWWWWWDLDLWWDLLLWLDDDWWWDLLLDLLWD
DLWWWLLLDLDLWWWWDLWWDWLLDDDDLWWLWWDLWDDDLWWDDWLDDLWWDDDDDDD
DLDDLWWWDLWLLLWLLDDWDWWLDLDLDLDWDDLDLLDWWLLWLLLWDDDWLWLLLLLD
WDLDWWDLWDLLDWLLDDDDLLLWWLWLLWLWWWLLWDLWLDLDWLWDDWLWDWLDWDLLD
DDLWLLLWLDLLDLLLWLWWLDLWDWLWLLWWWWDLWDLLWWWLWWDDDWDLWDWWWLDDLD
DDLWDWWDWDLLDLLWLDWDDWWLDWDWLLWWLWWDLDLLDDDLLDWWLWLDDLWWLDDDDLD
DWDDDDLDLLWDWWWDWDDWDDWWWWDWLWDWWLDWDLDWWWDLDDDLDLLWLLLDDDWDLDDD
WLLWLWWWWLLLWDWWDDLLDWLDWLWWWDWDDLWLWWWDLLLDWWWLDWWWLWDDWLDLLDLWD
DLLDLWWWWDLDLLLWWLWWDWWLDDDLDWWDDLDDLDDLLLWLDWDLDWLWDDDDWLDWWDLDLD
LLDWDWDDWLWDWLLDWLDDWLLLLWWLWLLDDWLDWDDWDDLLWLDDLLLLDLLLWLLLWLLDWWD
DDLDWLLDWWWDLDLDDWWWWWLDLLLLDLLDWLWWLLLWDLWDWLWLDLLWLLDDDDWWWLDDWWDD
LLWWWWWDDWLDDWLDLWWDLLDLLWWLWWLWWLWDWLWLLLDLDWDDLLWDLDDWWWLDLDLWWWDWD
LLLDDDWDLDLLDWLWDWWLWWLDDWLDWLDLWLLWLDWLDDWWWLLWDDWDWWLDLDLDWWDLLDWDWD
LLLDWDDWLLDDLDLDWDLLWDDDWDWDDLWDLWWLDDLLDWDDDDLLDLLDLLWDDDWLWDWWWWWWDWD
DLDLDWLWDDWLDWWLDDLLLLWLLDWDLWDWWWWLLWWDLLWDLWDDWWWDWWLWWLWWWWLWWDWLDLLD
DWLDLWWWWDLLWDDLWLLWDLWLDWDLWDWLDLDWDDDWDWLWDLDWWLWDLWLDDLWDDDLWWLLLLDDLD
LDDLWWDDWLLLDLWWLDDLWDLLLDDDDWDLLLLWLLLWLDWLWWWWLWWDWDWLDLLLDLWDLWWWWWDDLD
WDWDLWDWWLLWWWWDLLDLDWWLDDLDDWWWDWDLLLLWDDDDDWDDLDLWWWWDLWWLLLDLLDWWWWDDLDD
DDWLWWDDLDWDWLDDDDWLLWLDWLWLWLLLDWDDDWLWWWDLLLDLWDLLLLWWWLLDLWLDWWLWDLWWLDWD
WWWWLLWDLWDDWWDWDWDLDLLLLWWWDWDDWWDWLDDLWLDLLDDWWWWDDLDDWDWDDDLLDDDWDLDWWLLLD
LLLLLDDLWDWDDDLDLWDDLLWWDDWDWDWLLWLDLDWDLLLDLLDLWDDWDDDDWLLWWDWWWWWLLLDDDLDDWD
WWLDLLLDLWLDDLLDDDWDDLWLDWLWLDDLDWLLDWDDWDLDWLLDLDDLDLLWWDWWDWDWWDLWLDDDLDLWDWD
WWDLDLWWWLDDDDWWDWLWDLDDWWWWWWLWWWDWLDDWDLDWLWWWDWDDWDWWWLLDLWDDDDDWWDLWWLWDWDDD
LLLWLLLWLDDLLLDLLDWWLLWWDLDDDLWLLWLLWWWLWDDLWLLWWWWWWDDWDLLWDWLLLLDDDWDDWWDDLLLWD
DWLDWWDWDWWDLDWLLWDDDDWWDLDWLWDLWLWWLDLLDWWWLDLDDWDLDWWLLLLDWDWDLDWDLLDDDLLWDDLWWD
WDWWLDWLWDDLDDLDWLDLDLLWDWDDWDWDLLLWDLWLDWWWWLWWDLDDDWLWLDDLLLDLWDDDDDWDWDLLDWWDLDD
LWDDDLWLLLDWWLWWWDWDDWWWLWLLDDLWDWLDDLWLDWWDWWLWLLWLLWWWDLLWWDDWLWLDLWWWDLLWWDLDLLWD
WLDLWWWWDWWWLDWDLWLLLDLDDWLDWLDDDDLWDDWDWWDWDLDDWWLDDDWDDLLWDLWDWWLLWDLDLDWLLLLWLDWDD
LWWWDLLLWWLWLLLWDLDLWLWDDLDLDDDWWDDDDWLDWLLWWWDLWLWLLWWLLWLLDWLLDDWWDLDLWDLLDWWDDDLDLD
WLLDDWDDLWLLDDLWDLWWLWWDDLWDLDDWLWLWDDWLWWLDWWLDWDWWWDWDDLLDDLWDDLWDDDDWDWLDDWDDLWDLLLD
DLDLDWLLDWLDWWLLDDDLLDWLDWLLWWDWDDDLWDDWLDDDWLWWLWLDLLWLDLWWWDLDWLDWDLLWDWWDLDWLWWWLLWLD
LDDWLDLWWWWWWDDDDWDDWLWDWDWDWWLLWDDDWDLDDDDWDDLLDWLDWDWDWDLLLLDDLWLDDDLWLDWLWWDWLDLWWWLDD
WLWLWWLLWDWLLLDWWDDLDWWDWLDLLWLDLLDWDDDLWDWDDDWWDDLDLWDWDLDDDDDWLWWWWWLLLLLLWLDDDWLWWWDDLD
LDWWLDLLDLLDWWWDDDLWWDLWWLLDDDWWLDLDWDWWLLDLDWLWWDWWDWLLWDLLWLDDLLWWDLWDDWWDDDWDDWLDWLWLDDD
DWLLWWDWDWDWWWWWWDDWWLDLWLDWLWLLLLWWDWDDLLLDDDLWWDLLWLDWWDDDWDLWLDDWDDLDWWWLWWWLLLLWWLWDWLDD
LWLLLDDWLWWWDWDLDLDDWWDLWLWWDLWLDLWLLDDWDWWWLDLWDDDWDDLWLLWLDWWLDWWDDLDDDLLWDWLDWDWDWWLLDDLDD
LLDLLWLLLDWLDDLLWDDWDDWDDLLLLLLLWLDLWLLLWLDLDLDDLWWWLLDWWDLDDLLWLWDDLWWDDWLDWWWDDWLLDWWLLDWWWD
LWDDDDLDDDLLLDWLDLLDWLWLLWWDLDLLLDLLDDWLWWDWWLLDWLWWLDLDDWWDWWLWWLDLWLDLDWDWLDWDWLLWWLWDLDWDDWD
DLLLDDLWWDLDLWDWWDDLLLWWWWDWWWLDLDDLLDWWDWWWDLWLLLWDLWDWLLLLWWDDDDWWDLDLLLDLWLDDLDDDDLWDDLDWWWDD
LLLDLLDLWDWWDWWDLDWDLWWLLDWLLDDLWDLWLLLDWLLWDWLLDWDLDWDLLDWWLDWLWWDDDWDWLDLLLWDWDWWLLWWWDLDWLLLLD
1 50
4 16
20 5
82 77
96 78
60 60
30 26
31 15
6 72
56 17
31 59
9 66
64 17
14 83
10 25
82 86
8 49
43 50
17 49
9 62
80 40
33 80
89 39
28 86
59 75
17 36
78 87
48 76
63 41
9 80
35 34
9 26
23 43
39 55
59 79
84 95
29 32
56 60
40 17
29 6
59 40
95 71
91 48
16 73
80 59
40 86
51 5
90 25
69 65
88 28
4 34
40 86
43 30
4 12
52 90
56 71
7 8
66 52
60 69
66 64
84 74
14 40
43 51
54 12
89 46
38 17
24 9
4 85
15 31
70 96
25 20
45 15
18 32
94 91
7 13
62 19
97 5
52 81
67 89
67 72
84 74
78 40
50 41
13 62
29 69
97 25
42 43
12 24
69 4
47 83
48 11
96 63
27 83
42 25
