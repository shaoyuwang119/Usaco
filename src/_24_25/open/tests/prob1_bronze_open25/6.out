1665
2184
1128
1488
2520
5688
2353
1488
2013
1665
1665
2520
1840
2013
2184
2353
1665
1309
573
1665
1488
1309
1309
1309
1309
1840
2353
2848
2520
1309
1309
2184
1665
945
2013
1840
1309
1840
1665
2520
2184
1840
1840
1840
760
3009
1309
2013
1840
1665
1840
3009
2520
1665
1488
2013
1665
1665
1665
2013
2353
2184
2013
1840
1488
1309
1309
2013
1488
2013
2353
2013
1665
2520
2013
2848
1665
2520
1665
1840
2353
2520
1665
1840
1488
2520
2184
1128
2184
1309
2184
1488
1840
2520
